__tests__/Utils/getQueryStringFromUrl.test.ts

```diff
@@ -0,0 +1,37 @@
+import { NextRouter } from 'next/router';
+
+import { getQueryStringFromUrl } from '@/utils/getQueryStringFromUrl';
+
+describe('getQueryString', () => {
+    const mockRouter: NextRouter = {
+        route: '/',
+        pathname: '/',
+        query: {
+            q: 'search:dark mode',
+        },
+        asPath: '/',
```

Comment:
I'm not sure about this, but is the path `/` or `/issues`.
The test won't be affected by this, but we should be testing on the route this function is supposed to work with.

Reply:
Done.